### PR TITLE
Adding index.html.

### PR DESCRIPTION
**Pull request by BritneyTP**
Here is my html for the august lab.

### CODE REVIEW

**Comment by Fiona0516** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <img src="http://i639.photobucket.com/albums/uu117/wandersick/email-icon-16x16.gif">
+    <img src="http://files.softicons.com/download/toolbar-icons/iconza-grey-icons-by-turbomilk/png/16x16/print.png">
+    <img src="http://files.softicons.com/download/application-icons/mega-pack-icons-1-by-nikolay-verin/png/16/Gmail.png">
+
+   <h2> Rain, thunderstorms developing from southern Plains to mid-Atlantic
+   </h2>
+   <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Repellat odio, inventore! Officia aut veniam aliquam dolorem, quod fugiat molestiae voluptates in numquam, eos tenetur, natus ipsam nobis repellat repudiandae ex.</p>
+    <a href="#">Read More...</a>
+
+  </main>
```
Comment (goodbye-internet) [inline index.html:47]: How about a footer?

Comment (BritneyTP) [inline index.html:47]: THERE IS NO FOOTER!!! lol

Comment (goodbye-internet) [inline index.html:47]: Fine. I'll approve.

Comment (BritneyTP) [inline index.html:47]: I cut my foot off lol

**Comment by goodbye-internet** [comment]
👍

**Pull request merged**